dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/cluster/ClusterManager.java

```diff
@@ -48,9 +50,13 @@ public class ClusterManager {
     @Autowired
     private RegistryClient registryClient;
 
+    private final WorkerGroupTaskDispatchManager workerGroupTaskDispatchManager;
+
     public ClusterManager() {
         this.masterClusters = new MasterClusters();
         this.workerClusters = new WorkerClusters();
+        workerGroupTaskDispatchManager = new WorkerGroupTaskDispatchManager();
```

Comment:
Can this work? `WorkerGroupTaskDispatchManager` is a spring bean, did you test this?